search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/ServiceTransformers.java

```diff
@@ -227,7 +236,7 @@ public static void initializeTransformers(
     mappers.put(VariableSort.class, new VariableFieldSortingTransformer());
     mappers.put(AuthorizationSort.class, new AuthorizationFieldSortingTransformer());
     mappers.put(RoleSort.class, new RoleFieldSortingTransformer());
-    mappers.put(TenantSort.class, new TenantFieldSortingTransformer());
```

Comment:
This was a duplicate addition that I removed. See line 232 for where it is added